routes/cube_routes.js

```diff
@@ -737,8 +739,9 @@ router.get('/list/:id', async (req, res) => {
       cardNames.push(card.details.name);
     });
 
-    const priceDict = await GetPrices([...pids]);
-    const eloDict = await getElo(cardNames, true);
+    const priceDictQ = GetPrices([...pids]);
```

Comment:
we should just pass `pids` into here

Reply:
I believe that didn't work because pids is a Set and GetPrices only works with an array.